staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5299,10 +5301,10 @@ type DownwardAPIVolumeFile struct {
 	// (limits.cpu, limits.memory, requests.cpu and requests.memory) are currently supported.
 	// +optional
 	ResourceFieldRef *ResourceFieldSelector `json:"resourceFieldRef,omitempty" protobuf:"bytes,3,opt,name=resourceFieldRef"`
-	// Optional: mode bits to use on this file, must be a value between 0
-	// and 0777. If not specified, the volume defaultMode will be used.
-	// This might be in conflict with other options that affect the file
-	// mode, like fsGroup, and the result can be other mode bits set.
+	// Optional: mode bits to use on this file, must be a decimal equivalent of
+	// octal value between 0 and 0777. If not specified, the volume
+	// defaultMode will be used. This might be in conflict with other options
+	// that affect the file mode, like fsGroup, and the result can be other mode bits set.
```

Comment:
```suggestion
	// Optional: mode bits used to set permissions on this file, must be an octal value between 0
	// and 0777. If not specified, the volume defaultMode will be used.
	// This might be in conflict with other options that affect the file
	// mode, like fsGroup, and the result can be other mode bits set.
```